src/dosbox.cpp

```diff
@@ -647,7 +654,7 @@ void DOSBOX_Init(void) {
 	        "  MIDI ports, and select one (for example 'midiconfig=14:0'\n"
 	        "  for sequencer client 14, port 0).\n"
 #endif
-	        "- When using a Roland MT-32 rev.0 as midi output device, some\n"
+	        "- When using a Roland MT-32 rev.0 as MIDI output device, some\n"
```

Comment:
Maybe we should specify: "a physical Roland MT-32 (rev.0) device"?

Reply:
Fixed in 86d25752